capabilityquerier/pkg/config/config.go

```diff
@@ -0,0 +1,94 @@
+package config
+
+import "github.com/spf13/viper"
+
+// SetupConfig associates the application with all of the relevant configuration parameters
+// for the application with the prefix 'lantern'.
+func SetupConfig() error {
+	var err error
+
+	viper.SetEnvPrefix("lantern")
+	viper.AutomaticEnv()
+
+	err = viper.BindEnv("quser")
+	if err != nil {
+		return err
+	}
+	err = viper.BindEnv("qpassword")
+	if err != nil {
+		return err
+	}
+	err = viper.BindEnv("qhost")
+	if err != nil {
+		return err
+	}
+	err = viper.BindEnv("qport")
+	if err != nil {
+		return err
+	}
+	err = viper.BindEnv("capquery_qname")
+	if err != nil {
+		return err
+	}
+	err = viper.BindEnv("capquery_numworkers")
+	if err != nil {
+		return err
+	}
+	err = viper.BindEnv("capquery_qryintvl") // in minutes
```

Comment:
Should add README with info on these new .env variables and add them to env.sample and .travis.yml if needed

Reply:
added readme. the unit tests don't use these variables, so don't need to be on .travis.yml. the service is not yet dockerized, so doesn't yet need to be in env.sample.